recipe/run_test.py

```diff
@@ -40,9 +40,11 @@
 driver = ogr.GetDriverByName('GML')
 assert driver is not None, 'GML not available within GDAL'
 
-# Only available when openjpeg successfully linked in.
-driver = gdal.GetDriverByName('JP2OpenJPEG')
-assert driver is not None, 'JP2OpenJPEG not available within GDAL'
+import sys
+if not sys.version_info >= (3, 4):
```

Comment:
All the comment I made on the code above will be on a new issue once this gets merged.